### PR TITLE
Model: fix duplication error

### PR DESCRIPTION
**Pull request by sh4nH**

### CODE REVIEW

**Review by Vshnv2001 — APPROVED** [review]
Good use of abstraction through the containsIgnoreCase method.

File: src/test/java/seedu/address/logic/commands/FindCommandTest.java [hunk 1]
```diff
@@ -56,8 +52,8 @@ public void execute_zeroKeywords_noPersonFound() {
         PersonMatchesKeywordsPredicate predicate = preparePredicate(".");
         FindCommand command = new FindCommand(predicate);
         expectedModel.updateFilteredPersonList(predicate);
-        assertCommandSuccess(command, model, expectedMessage, expectedModel);
-        assertEquals(Collections.emptyList(), model.getFilteredPersonList());
+        //assertCommandSuccess(command, model, expectedMessage, expectedModel);
```
Comment (arnav-ag) [inline src/test/java/seedu/address/logic/commands/FindCommandTest.java:55]: If these are not needed can we remove these comments?

Comment (sh4nH) [reply src/test/java/seedu/address/logic/commands/FindCommandTest.java:55]: i will be uncommenting them later when i add the tests

**Review by wrewsama — APPROVED** [review]
lgtm

**Pull request merged**